ci-operator/step-registry/assisted/baremetal/setup/assisted-baremetal-setup-ref.yaml

```diff
@@ -11,6 +11,8 @@ ref:
     name: edge-group-assisted-installer-bot
     mount_path: /var/run/assisted-installer-bot
   dependencies:
+  - name: "release:latest"
```

Comment:
be aware it means ``latest`` is a required dependency right now to the step, meaning all other components need to be updated if they don't have it
just mentioning that so that we'll do a big-enough list of rehearses to test all cases

There's another alternative to use ``OPENSHIFT_VERSION=4.11`` which shouldn't have this kind of problem afaik